Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -2144,7 +2144,8 @@ private void loadconfig()
             QualitySettings.vSyncCount = 1;
         }
         AudioListener.volume = PlayerPrefs.GetFloat("vol", 1f);
-        QualitySettings.masterTextureLimit = PlayerPrefs.GetInt("skinQ", 0);
```

Comment:
This was how settings were previously stored and loaded upon opening the game. Since you have implemented a new way of doing it, this line is no longer required.

Reply:
If you marked this as resolved, then also remove the two lines of code
`      //IS THIS LINE OF CODE IMPORTANT?? IT AFFECTS GRAPHICS SCRIPTS!!
        //QualitySettings.masterTextureLimit = PlayerPrefs.GetInt("skinQ", 0);`